net/quagga/src/opnsense/service/templates/OPNsense/Quagga/zebra.conf

```diff
@@ -0,0 +1,21 @@
+!
+! Zebra configuration saved from vty
+!   2017/03/03 20:21:04
+!
+!
+!
+!
+{% if helpers.exists('OPNsense.quagga.ospf.interfaces.interface') %}
+{%   for interface in helpers.toList('OPNsense.quagga.ospf.interfaces.interface') %}
+{%     if interface.enabled == '1' %}
+interface {{ interface.interfacename }}
+{%     endif %}
+{%   endfor %}
+{% endif %}
```

Comment:
There is one "!" between the interfaces and I think it is better to omit this settings as long as they are unused. Also Zebra should not use the `OSPF` settings. I think it is also valid to have a configuration file without interfaces inside.